uk.co.compucorp.civicrm.hrleaveandabsences/views/my-leave/components/my-leave-request.html

```diff
@@ -0,0 +1,177 @@
+              <div class="form-group">
+                <label class="col-xs-4 control-label">From Date
+                  <i class="fa fa-asterisk"></i>
+                </label>
+                <div class="col-xs-4">
+                  <div class="input-group chr-datepicker">
+                    <input type="text"
+                           class="form-control"
+                           uib-datepicker-popup
+                           show-button-bar="false"
+                           datepicker-options="{startingDay: 1,showWeeks: false}"
+                           ng-model="modal.leaveRequest.fromDate"
+                           is-open="modal.leaveRequest.showDatePickerFrom"/>
+                    <span class="input-group-btn">
+                        <button type="button" class="btn btn-default fa fa-calendar"
+                                ng-click="modal.leaveRequest.showDatePickerFrom = true"></button>
+                      </span>
+                  </div>
+                </div>
+                <div class="col-xs-4">
+                  <div class="chr_custom-select chr_custom-select--full">
+                    <select>
+                      <option value="1" selected>1/2 - PM</option>
+                      <option value="2">1/2 - AM</option>
+                    </select>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-xs-12 form-horizontal">
+              <div class="form-group">
+                <label class="col-xs-4 control-label">To Date
+                  <i class="fa fa-asterisk"></i>
+                </label>
+                <div class="col-xs-4">
+                  <div class="input-group chr-datepicker">
+                    <input type="text"
+                           class="form-control"
+                           uib-datepicker-popup
+                           datepicker-options="{startingDay: 1,showWeeks: false}"
+                           show-button-bar="false"
+                           ng-model="modal.leaveRequest.toDate"
+                           is-open="modal.leaveRequest.showDatePickerTo"/>
+                    <span class="input-group-btn">
+                        <button type="button" class="btn btn-default fa fa-calendar"
+                                ng-click="modal.leaveRequest.showDatePickerTo = true"></button>
+                      </span>
+                  </div>
+                </div>
+                <div class="col-xs-4">
+                  <div class="chr_custom-select chr_custom-select--full">
+                    <select>
+                      <option value="1" selected>1/2 - PM</option>
+                      <option value="2">1/2 - AM</option>
+                    </select>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <hr/>
+          <div class="row form-group">
```

Comment:
Here you are using `.form-group` in order to have the margin bottom, but it is semantically incorrect, as you do not have any form controls.

Why couldn't you have something like `.chr_leave-request-form` applied to the whole modal body, which you can then use to give layout rules? You could have then elements like `.chr_leave-request-form__balance` which then defines the layout rules of this section